src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.csproj

```diff
@@ -33,6 +33,7 @@
       - .NET 6.0
       - .NET 6.0 Windows.18362+
       - UWP 10.0.16299
+	    - UWP 10.0.17763 with .NET 9
```

Comment:
nit: Indentation is different

Reply:
I'm struggling to fix this in my editor. If you can send a workspace commit I'll update it.